doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -206,7 +206,11 @@ nRF9160 samples
 
   * Renamed. The previous name was nRF9160: GPS with SUPL client library.
   * Added support for nRF Cloud A-GPS and P-GPS.
-  * LTE now remains connected to the network all the time when assistance is enabled. You can enable the old behavior with A-GPS using a Kconfig option.
+  * LTE now remains connected to the network all the time when assistance is enabled.
+    You can enable the old behavior with A-GPS using a Kconfig option.
```

Comment:
**nitpick**: document what the "old" behaviour is, and which Kconfig option enables it.

Reply:
I'll change this although it's not part of this PR. Just fixed the sentence to start on a new line. :)